CourseApp.Tests/ProgramTest.cs

```diff
@@ -0,0 +1,139 @@
+namespace CourseApp.Tests
+{
+    using Xunit;
+
+    public class ProgramTest
+    {
+        [Fact]
+        public void TestTaskOne1()
+        {
+            var a = 2.0;
+            var b = 3.0;
+            var x = 0.11;
+            var expected = 1.582;
+            Calculate calculateTask = new Calculate();
+            var actual = calculateTask.CalculateTask(a, b, x);
+            Assert.Equal(actual, expected);
```

Comment:
Please, write first expected value and then actual
According to xUnit docs: `Assert.Equal(expected, actual)`

Reply:
Fixed